Update SECURITY.md (#15502)
solana-labs/solana: SECURITY.md

<a name="bounty"></a>
## Security Bug Bounties
We offer bounties for critical security issues. Please see below for more details.

Loss of Funds:
$500,000 USD in locked SOL tokens (locked for 12 months)
* Theft of funds without users signature from any account
* Theft of funds without users interaction in system, token, stake, vote programs
* Theft of funds that requires users signature

Consensus/Safety Violations:

> $250k USD in locked SOL tokens (locked for 12 months)

Reviewer: ```suggestion
$250,000 USD in locked SOL tokens (locked for 12 months)
```

* Consesnus safety violation
* Tricking a validator to accept an optimistic confirmation or rooted slot without a double vote, etc..

Other Attacks:

>  $100k USD in locked SOL tokens (locked for 12 months)

Reviewer: ```suggestion
 $100,000 USD in locked SOL tokens (locked for 12 months)
```

>  $100k USD in locked SOL tokens (locked for 12 months)
> * Protocol liveness attacks, 
> * Eclipse attacks, 

Reviewer: ```suggestion
 $100k USD in locked SOL tokens (locked for 12 months)
* Protocol liveness attacks,
* Eclipse attacks,
```

* Remote attacks that partition the network,

>  

Reviewer: ```suggestion

```

DoS Attacks:

>  $25k USD in locked SOL tokens (locked for 12 months)

Reviewer: ```suggestion
 $25,000 USD in locked SOL tokens (locked for 12 months)
```

> * Non-RPC DoS, 

Reviewer: ```suggestion
* Non-RPC DoS,
```

* Remote resource exaustion via Non-RPC protocols

>  $5k

Reviewer: ```suggestion
 $5,000 USD in locked SOL tokens (locked for 12 months)
```

>  $5,000 USD in locked SOL tokens (locked for 12 months)

Reviewer: @scriptrunner2049 - this part is a little jumbled up, are these two separate sections?

Author: Good pick-up. I've edited the doc.

RPC DoS/Crashes:

> $5k USD in locked SOL tokens (locked for 12 months)

Reviewer: ```suggestion
$5,000 USD in locked SOL tokens (locked for 12 months)
```

Eligibility:
* The participant submitting the bug bounty shall follow the process outlined within this document
* Valid exploits can be eligible even if they are not successfully executed on the cluster
* Multiple submissions for the same class of exploit are still eligible for compensation, though may be compensated at a lower rate, however these will be assessed on a case-by-case basis

> * Participants must complete KYC and sign the participation agreement here when the registrations are open https://solana.com/validator-registration. Security exploits will still be assessed and open for submission at all times. This needs only be done prior to distribution of tokens. 

Reviewer: ```suggestion
* Participants must complete KYC and sign the participation agreement here when the registrations are open https://solana.com/validator-registration. Security exploits will still be assessed and open for submission at all times. This needs only be done prior to distribution of tokens.
```

> Notes: 

Reviewer: ```suggestion
Notes:
```
